src/Http/Http/src/SendFileFallback.cs

```diff
@@ -3,13 +3,16 @@
 
 using System;
 using System.IO;
+using System.IO.Pipelines;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.AspNetCore.Http
 {
     public static class SendFileFallback
     {
+        const int DefaultBufferSize = 1024 * 16;
```

Comment:
Remove this.

Reply:
This was being used in `FileStream` and `StreamCopyOperationInternal.CopyToAsync`.

Reply:
I know, but we don't need it here.

Reply:
So I move it to `SendFileAsync(Stream destination,...` and `GetFileStream` methods, right?